src/setup.py

```diff
@@ -69,13 +74,18 @@
 
 # 3
 gameFolder = os.environ['PROGRAMFILES(X86)'] + "\\Steam\\steamapps\\common\\Hearts of Iron IV"
-try:
-    os.listdir(gameFolder)
-except Exception as e:
-    print(e)
-    # 3.2
-    print("Can't find the game path, please enter the path manually: ")
-    gameFolder = input() # TODO: Check if the path is valid
+while True:
+    try:
+        if "Hearts of Iron IV.exe" not in os.listdir(gameFolder):
```

Comment:
```suggestion
        if "hoi4.exe" not in os.listdir(gameFolder):
```

The executable file is named as: "hoi4.exe"